rir/src/compiler/debugging/stream_logger.cpp

```diff
@@ -198,5 +193,29 @@ void LogStream::section(const std::string& title) {
     highlightOff();
 }
 
+void StreamLogger::beginPhase(const std::string& msg) {
+    if (!options.includes(DebugFlag::PrintIntoFiles) &&
```

Comment:
Why the phase is not included if we print into files?

Reply:
which file? There is one file per function. But the phase is a global thing...